src/main/java/it/gov/pagopa/pu/send/service/SendServiceImpl.java

```diff
@@ -72,4 +79,19 @@ public void uploadFiles(String sendNotificationId) {
     }
     sendNotificationRepository.updateNotificationStatus(sendNotificationId, NotificationStatus.UPLOADED);
   }
+
+  @Override
+  public void deliveryNotification(String sendNotificationId) {
+    SendNotification notification = sendNotificationRepository.findById(sendNotificationId)
+      .orElseThrow(() -> new IllegalArgumentException(NOTIFICATION_NOT_FOUND + sendNotificationId));
```

Comment:
instead of create a constant for the common message, why don't you create a common method `findSendNotification`?